extensions/agroal/deployment/src/test/java/io/quarkus/agroal/test/DefaultDataSourceConfigTest.java

```diff
@@ -35,12 +35,13 @@
     @Test
     public void testDefaultDataSourceInjection() throws SQLException {
         testDataSource(defaultDataSource, "username-default", 3, 13, 7, Duration.ofSeconds(53), Duration.ofSeconds(54),
-                Duration.ofSeconds(55), Duration.ofSeconds(56), Duration.ofSeconds(57));
+                Duration.ofSeconds(55), Duration.ofSeconds(56), Duration.ofSeconds(57),
+                "create schema if not exists schema_default");
     }
 
     private static void testDataSource(AgroalDataSource dataSource, String username, int minSize, int maxSize,
             int initialSize, Duration backgroundValidationInterval, Duration acquisitionTimeout, Duration leakDetectionInterval,
-            Duration idleRemovalInterval, Duration maxLifetime) throws SQLException {
+            Duration idleRemovalInterval, Duration maxLifetime, String initialSQL) throws SQLException {
```

Comment:
It's very minor but let's be consistent and call the variable `initialSql`.

Reply:
Sure